tests/acceptance/constants.js

```diff
@@ -1,2 +1,10 @@
-module.exports.VERTICAL_SEARCH_URL_REGEX = /v2\/accounts\/me\/answers\/vertical\/query/;
-module.exports.UNIVERSAL_SEARCH_URL_REGEX = /v2\/accounts\/me\/answers\/query/;
+export const VERTICAL_SEARCH_URL_REGEX = /v2\/accounts\/me\/answers\/vertical\/query/;
+export const UNIVERSAL_SEARCH_URL_REGEX = /v2\/accounts\/me\/answers\/query/;
+export const PORT = 9999;
+
+export const UNIVERSAL_PAGE = `http://localhost:${PORT}/tests/acceptance/fixtures/html/universal`;
```

Comment:
we can extract the folder prefix to a variable, which is a little extra but a little more concise as well